libgalois/src/analytics/pagerank/pagerank-pull.cpp

```diff
@@ -40,12 +41,11 @@ InitNodeDataTopological(Graph* graph) {
   katana::do_all(
       katana::iterate(*graph),
       [&](const GNode& n) {
-        auto& sdata_value = graph->GetData<NodeValue>(n);
-        auto& sdata_nout = graph->GetData<NodeNout>(n);
-        sdata_value = init_value;
-        sdata_nout = 0;
+        auto& sdata = graph->GetData<NodeValueOut>(n);
+        sdata.value = init_value;
+        sdata.out = 0;
       },
-      katana::no_stats(), katana::loopname("initNodeData"));
+      katana::loopname("initNodeData"));
```

Comment:
Why are you enabling stats? I'm fine with it. I just don't want to do it by accident.

Reply:
It's intentional. I requested Hochan to enable stats for all our `do_all`s so that we can have a better idea of where we're seeing performance differences.